app/services/medium_providers/provider.rb

```diff
@@ -13,7 +13,7 @@ def self.friendly_name
       name
     end
 
-    class Jail < Safemode::Jail
+    class Jail < ApplicationRecord::Jail
```

Comment:
This is imho wrong, we do not inherit from `ApplicationRecord`, so we should not inherit it's Jail :thinking: